platforms/android/library/src/androidTest/java/io/element/android/wysiwyg/EditorStyledTextViewTest.kt

```diff
@@ -34,6 +35,29 @@ internal class EditorStyledTextViewTest {
             .check(matches(withText("Hello, world")))
     }
 
+    @Test
+    fun testUrlClicks() {
```

Comment:
Should we also add a test for setting a mention using `setHtml()` and then clicking it? It shouldn't block this PR but noticed it is missing in the [coverage report](https://app.codecov.io/gh/matrix-org/matrix-rich-text-editor/pull/895/blob/platforms/android/library/src/main/java/io/element/android/wysiwyg/EditorStyledTextView.kt).

Reply:
Added it in https://github.com/matrix-org/matrix-rich-text-editor/pull/895/commits/ed4b1ed53d78e4ccc549379283474e285d6e4f98. Let's see if it gets correctly picked by the coverage collector.